package.json

```diff
@@ -60,5 +60,9 @@
   "engineStrict": true,
   "engines": {
     "node": ">=10.10.0"
+  },
+  "dependencies": {
```

Comment:
Adding dependencies to the root package.json is probably not a good idea. We should scope these to the smoke tests project.